ocs_ci/templates/provider-client-deployment/storage_client_subscription.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: openshift-storage-client
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: openshift-storage-client-operator-group
+    namespace: openshift-storage-client
+spec:
+    targetNamespaces:
+    - openshift-storage-client
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: ocs-client-operator
+    namespace: openshift-storage-client
+spec:
+    channel: stable-4.14
```

Comment:
here should be placeholder instead of the channel, '{}'
we need to generate channel by version we use, dynamically

Reply:
Noted!!
I will address this in future in another pr as its not an immediate requirement for this release.

Reply:
Have addressed this.